### DISCUSSION TITLE
글자에서 줄바뀜이 먹혔으면 좋겠어요.

### DISCUSSION THREAD

**Discussion by endmoseung** [Yangjaecheon-Otter-Guardians/simple-thumbnail#119] [Ideas]
### 주제

Features

### Body

사용자가 enter를 누르면 <br>태그를 넣어준다던가 해서 줄바뀜이 가능했으면 좋겠어요!!

### BDD TASK

```bash
- [ ] 랜덤 이미지 배경 적용 후 다운로드시 정상적으로 이미지가 보인다.
...
```

**Comment by da-in** [comment]
오오! 좋은 아이디어 감사합니다!😮

그렇게 된다면
`(기존) H1 사이즈의 텍스트를 두 개 쓰는 것` == `(수정 후) H1 사이즈의 텍스트 하나에서 줄바꿈을 하는 것`

같은 효과를 볼 수 있는데, 텍스트 하나 안에서 해결되어 편리한 면이 있겠군요!
다들 논의해보면 좋을 것 같습니다🙂
